### ISSUE TITLE
[Android Bootcamp] Project 0: ToDo - Ready for review

### ISSUE THREAD

**Issue by adilansari** [adilansari/MeeDoo#5]
My app is complete, please review. Also, I took some reference for the gif provided by you, is that cool? /cc @codepathreview @codepath

**Comment by nesquena** [comment]
Adil,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer  learn the practical elements of Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

Great to see you were able to complete some extension tasks already as well. The extension tasks available on each project are often the most valuable learnings since they dive deeper into common real-world Android use cases. 

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and to **schedule a short 10 minute phone conversation** with us. Navigate to [today's date and onward from there](https://calendar.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose a 15-minute slot. Let us know if none of those times work.

Once you select a slot, can you make sure to **include best number** to reach you at to the invite? Look forward to chatting soon!

**Comment by nesquena** [comment]
> Also, I took some reference for the gif provided by you, is that cool? 

Of course, we like to see applicants that put time into creating a better user experience for the app. Would like to see you continue iterating on the UI/UX and/or functionality.
